plugins/loop_registry.go

```diff
@@ -45,14 +42,15 @@ func NewLoopRegistry(lggr logger.Logger, tracing config.Tracing, telemetry confi
 	}
 }
 
-// Register creates a port of the plugin. It is not idempotent. Duplicate calls to Register will return [ErrExists]
+// Register creates a port of the plugin. It is idempotent. Duplicate calls to Register will return the same port.
```

Comment:
Any possible unintended consequences?
Also why would anybody call it more than once?